src/frontend/components/Settings/sections/WeatherSettings.tsx

```diff
@@ -2,19 +2,33 @@ import { useState } from 'react';
 import { BaseSettingsSection } from '../components/BaseSettingsSection';
 import { WeatherWidgetSettings } from '../types';
 import { useDashboard } from '@irdashies/context';
+import { ToggleSwitch } from '../components/ToggleSwitch';
+// sortable list is not used right now — keep import commented until needed
+// import { useSortableList } from '../../SortableList';
 
 const SETTING_ID = 'weather';
 
 const defaultConfig: WeatherWidgetSettings['config'] = {
   background: { opacity: 0 },
+  includeAirTemp: true,
+  includeTrackTemp: true,
+  includeWind: false,
+  includeWetness: false,
+  includeTrackState: false,
```

Comment:
Can we use the same pattern for the enabled state so its a nested property.

e.g. 
```
          "airTemp": {
            "enabled": true
          },
          "trackTemp": {
            "enabled": true
          },
          "wetness": {
            "enabled": true
          },
```
Also why not have them all enabeld by default?